src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -237,6 +300,12 @@ export default {
 				checked: val => val,
 			};
 		}
+		if (this.enableRadioBtnExperiment) {
+			validations.selectedComms = {
+				required,
+				checked: val => val !== '',
```

Comment:
Interesting, so we're making them click one more time before submitting their order? This is a new click, right?